src/main/resources/application.yml

```diff
@@ -0,0 +1,45 @@
+server:
```

Comment:
As I mentioned in `src/main/java/com/cms/users/entity/UserBase.java`, I would spec to see the secrets here or just a placeholder for environment variables

Reply:
Thank you for this observation next release will include this property